__tests__/client/initClient.spec.js

```diff
@@ -153,8 +153,8 @@ describe('initClient', () => {
 
     await initClient();
 
```

Comment:
Shouldn’t we update `initClient` as well to start using the new react approach?

Reply:
This is using the new approach. https://github.com/americanexpress/one-app/pull/1212/files#diff-cb26e299c3c333198ccd95e53e4f18369a1268c72074277b91d0819b309f122bR73-R79

If you are referring to renderToPipeableStream thats a huge rewrite that would be part of another PR. its also not compatible with Fastify out of the box.

Reply:
gotcha